calico-enterprise/operations/cnx/configure-identity-provider.mdx

```diff
@@ -74,6 +74,37 @@ When configuring your cluster, you may be asked for the following inputs:
      clientSecret: <your-base64-clientid-secret>
    ```
 
+   If you are using external secret provider, you can mount your OIDC credentials as a volume through CSI Secret Store
+   Driver to avoid storing the credentials in the Kubernetes cluster.
+
+   Create a `SecretProviderClass` named `tigera-oidc-credentials` in the `tigera-operator` namespace that mounts your
+   OIDC credential fields as exactly two files: `clientID` and `clientSecret`.
+
+   In its spec, map your source secret/keys so the mounted filenames are precisely `clientID` and `clientSecret`. Use
+   your provider’s standard configuration options to wire up the values for those two files, and verify the resulting
```

Comment:
1. Not sure what you mean by 'wire up'. Can you clarify?

2. "verify the resulting volume contains only `clientID` and `clientSecret`."
   I don't quite understand. Are you saying that you want the volume to contain only these two files? As in, no other files?